### ISSUE TITLE
Symfony\Component\Console\Helper\HelperSet->get('table')

### ISSUE THREAD

**Issue by netengine-at** [liebig/cron#79]
Hi,

we have just updated our App to Laravel 5.2 and we are now getting an exeption.

> /vendor/liebig/cron/src/Liebig/Cron/RunCommand.php(51): Symfony\Component\Console\Helper\HelperSet->get('table')

I found this in google:

> The Table Helper was deprecated in Symfony 2.5 and removed in Symfony 3.0. You should now use the Table class instead which is more powerful.

Thanks in advance!

**Comment by liebig** [comment]
Hi,

yes, we had a similar problem with isue #77. It seems, that Laravel uses Symfony 3 in the new 5.2 version.  Can you remove the lines 51-56 of the /vendor/liebig/cron/src/Liebig/Cron/RunCommand.php and add the following code:
```php
$table = new \Symfony\Component\Console\Helper\Table($this->getOutput());
$table
    ->setHeaders(array('Run date', 'In time', 'Run time', 'Errors', 'Jobs'))
    ->setRows(array(array($report['rundate'], $inTime, round($report['runtime'], 4), $report['errors'], count($report['crons']))));
$table->render();
```

If this works, I will add the code.

Thank you!

**Comment by netengine-at** [comment]
Hi,

Wow, thanks for the super fast response! Yes, that did the trick, no more exeption.

Thank you!

**Comment by liebig** [comment]
Bug should be fixed in version 1.1.1. If not, please feel free to reopen this issue. Thank you.
